pkg/util/export/etl/db/db_holder.go

```diff
@@ -330,11 +330,25 @@ func bulkInsert(ctx context.Context, sqlDb *sql.DB, records [][]string, tbl *tab
 	escapedCSVData := strings.ReplaceAll(csvData, "'", "''")
 	loadSQL := fmt.Sprintf("LOAD DATA INLINE FORMAT='csv', DATA='%s' INTO TABLE %s.%s", escapedCSVData, tbl.Database, tbl.Table)
 
-	// Execute the LOAD DATA command directly without a transaction
-	_, err := sqlDb.Exec(loadSQL)
+	// Begin a new transaction
```

Comment:
why?   Isn't this a single command, so auto commit mode works just fine?

Reply:
There are duplicates. I do not know where they are from, so added the tx here.  Still unknown...

Reply:
duplicate?   what is the error message?   the table does not have any primary key etc so where does duplicates come from?

Reply:
It does help. And retry helps even more.

Reply:
But after fixing escape issue, I do not see the duplicates anymore.  So may try to remove it to test

Reply:
Should be no problem now